nnpdf_data/nnpdf_data/commondata/ATLAS_WPWM_13TEV/metadata.yaml

```diff
@@ -37,34 +38,30 @@ implemented_observables:
       - W
     plot_x: ' '
   kinematic_coverage:
-  - k1
-  - k2
-  - k3
+  - m_W2
+  - sqrts
   kinematics:
     variables:
-      k1:
-        description: Variable k1
-        label: k1
-        units: ''
-      k2:
-        description: Variable k2
-        label: k2
-        units: ''
-      k3:
-        description: Variable k3
-        label: k3
-        units: ''
-    file: kinematics_TOT.yaml
+      m_W2:
+        description: W boson squared mass
+        label: m_W2
```

Comment:
I think the key 'label' is the one used in the plots, so it would be better to use latex here, *e.g.* $m_W^2$.

Reply:
Thanks for spotting this, had miss it